ProductService.js

```diff
@@ -0,0 +1,140 @@
+export async function getProductList(page, pageSize, keyword) {
+  let url;
+  let res;
+  try {
+    url = new URL("https://panda-market-api-crud.vercel.app/products");
+    url.searchParams.append("page", page);
+    url.searchParams.append("pageSize", pageSize);
+    if (arguments[2] !== undefined) {
+      url.searchParams.append("keyword", keyword);
+    }
```

Comment:
ArticleService에서 적으신 것처럼, 파라미터를 직접 체크하는 것이 더 명확합니다.
```suggestion
    if (keyword !== undefined) {
      url.searchParams.append("keyword", keyword);
    }
```